WARNING - Received unrecognized parameter ensemble_download_models_timeout_sec
Running on local machine
Parent Run ID: AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60

Current status: DatasetFeaturization. Beginning to featurize the dataset.
Current status: DatasetFeaturizationCompleted. Completed featurizing the dataset.
Heuristic parameters: Target_Lag = '[1]', Target_Rolling_Window = '2'.
/projappl/project_2003107/anaconda3/envs/azure_automl/lib/python3.7/site-packages/azureml/_restclient/workspace_client.py:380: DeprecationWarning: The 'warn' method is deprecated, use 'warning' instead
  self._logger.warn('Converting non-string tag to string: (%s: %s)', key, val)
WARNING - Converting non-string tag to string: (forecasting_target_lags: [1])
WARNING - Converting non-string tag to string: (forecasting_target_rolling_window_size: 2)
WARNING - Converting non-string tag to string: (forecasting_max_horizon: 2064)
Current status: DatasetCrossValidationSplit. Generating individually featurized CV splits.
Current status: DatasetFeaturization. Beginning to featurize the CV split.
Current status: DatasetFeaturizationCompleted. Completed featurizing the CV split.
Current status: DatasetFeaturization. Beginning to featurize the CV split.
Current status: DatasetFeaturizationCompleted. Completed featurizing the CV split.
Current status: DatasetFeaturization. Beginning to featurize the CV split.
Current status: DatasetFeaturizationCompleted. Completed featurizing the CV split.
Current status: DatasetFeaturization. Beginning to featurize the CV split.
Current status: DatasetFeaturizationCompleted. Completed featurizing the CV split.
Current status: DatasetFeaturization. Beginning to featurize the CV split.
Current status: DatasetFeaturizationCompleted. Completed featurizing the CV split.

****************************************************************************************************
DATA GUARDRAILS: 

TYPE:         Frequency detection
STATUS:       PASSED
DESCRIPTION:  The time series was analyzed, all data points are aligned with detected frequency.
              

****************************************************************************************************

TYPE:         Missing feature values imputation
STATUS:       PASSED
DESCRIPTION:  No feature missing values were detected in the training data.
              Learn more about missing value imputation: https://aka.ms/AutomatedMLFeaturization

****************************************************************************************************

TYPE:         Memory Issues Detection
STATUS:       PASSED
DESCRIPTION:  The selected horizon, lag and rolling window values were analyzed, and no potential memory issues were detected.
              Learn more about time-series forecasting configurations: https://aka.ms/AutomatedMLForecastingConfiguration

****************************************************************************************************
Current status: ModelSelection. Beginning model selection.

****************************************************************************************************
ITERATION: The iteration being evaluated.
PIPELINE: A summary description of the pipeline being evaluated.
DURATION: Time taken for the current iteration.
METRIC: The result of computing score on the fitted pipeline.
BEST: The best observed score thus far.
****************************************************************************************************

 ITERATION   PIPELINE                                       DURATION      METRIC      BEST
/projappl/project_2003107/anaconda3/envs/azure_automl/lib/python3.7/site-packages/azureml/automl/core/_downloader.py:81: DeprecationWarning: time.clock has been deprecated in Python 3.3 and will be removed from Python 3.8: use time.perf_counter or time.process_time instead
  download_start_time = time.clock()
/projappl/project_2003107/anaconda3/envs/azure_automl/lib/python3.7/site-packages/azureml/automl/core/_downloader.py:89: DeprecationWarning: time.clock has been deprecated in Python 3.3 and will be removed from Python 3.8: use time.perf_counter or time.process_time instead
  download_end_time = time.clock()
         0   /projappl/project_2003107/anaconda3/envs/azure_automl/lib/python3.7/site-packages/sklearn/base.py:197: FutureWarning: From version 0.24, get_params will raise an AttributeError if a parameter cannot be retrieved as an instance attribute. Previously it would return None.
  FutureWarning)
MaxAbsScaler DecisionTree                      0:06:38       0.7425    0.7425
         1   MinMaxScaler DecisionTree                      0:05:41       0.7430    0.7425
         2   StandardScalerWrapper LassoLars                0:03:18       0.7336    0.7336
         3   RobustScaler ElasticNet                        0:18:43       0.7337    0.7336
         4   WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_4/outputs/conda_env_v_1_0_0.yml?sv=2019-02-02&sr=b&sig=8TLk3jqxxWDYQIoFy6C18ZVzNfGc68A4b58EzqfIyDw%3D&st=2020-10-02T19%3A13%3A44Z&se=2020-10-03T19%3A23%3A44Z&sp=rcw&comp=block&blockid=TURBd01EQXdNREF3TURBd01EQXdNREF3TURBd01EQXdNREF3TURBd01EQSUzRA%3D%3D&timeout=30
StandardScalerWrapper LassoLars                0:03:20       0.7336    0.7336
         5   RobustScaler ElasticNet                        0:13:52       0.7336    0.7336
         6   RobustScaler DecisionTree                      0:06:00       0.7416    0.7336
         7   MinMaxScaler DecisionTree                      0:07:27       0.7419    0.7336
         8   RobustScaler DecisionTree                      0:06:34       0.7442    0.7336
         9   MinMaxScaler DecisionTree                      0:06:19       0.7434    0.7336
        10   StandardScalerWrapper DecisionTree             0:04:23       0.7413    0.7336
        11   WARNING - Received unrecognized parameter ensemble_download_models_timeout_sec
WARNING - Received unrecognized parameter ensemble_download_models_timeout_sec
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Connection pool is full, discarding connection: automlws9124637538.blob.core.windows.net
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_5/outputs/model.pkl?sv=2019-02-02&sr=b&sig=vO06RJE5bQ6Fc19%2FKBmaqqsMRyZL24RqdIbJVQ06eYQ%3D&st=2020-10-02T20%3A01%3A35Z&se=2020-10-03T04%3A11%3A35Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_10/outputs/model.pkl?sv=2019-02-02&sr=b&sig=VtpR50tazr7hcGBYA18ih2KaLFk91zCpeFmx880ok0Y%3D&st=2020-10-02T20%3A01%3A34Z&se=2020-10-03T04%3A11%3A34Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_4/outputs/model.pkl?sv=2019-02-02&sr=b&sig=7xagjGdS3jvM3kf2AlC8N8iS6fhOZNnnkl7QFlB0Ybo%3D&st=2020-10-02T20%3A01%3A35Z&se=2020-10-03T04%3A11%3A35Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_3/outputs/model.pkl?sv=2019-02-02&sr=b&sig=ZwdnFvx3Wcn7uQx1t6VCnx6gnDVEHBvH2mo0ghPY414%3D&st=2020-10-02T20%3A01%3A35Z&se=2020-10-03T04%3A11%3A35Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_10/outputs/model.pkl?sv=2019-02-02&sr=b&sig=VtpR50tazr7hcGBYA18ih2KaLFk91zCpeFmx880ok0Y%3D&st=2020-10-02T20%3A01%3A34Z&se=2020-10-03T04%3A11%3A34Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_4/outputs/model.pkl?sv=2019-02-02&sr=b&sig=7xagjGdS3jvM3kf2AlC8N8iS6fhOZNnnkl7QFlB0Ybo%3D&st=2020-10-02T20%3A01%3A35Z&se=2020-10-03T04%3A11%3A35Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_2/outputs/model.pkl?sv=2019-02-02&sr=b&sig=TM1T0V5C2xMB%2FaR6UxYzrk5%2FD%2BX1ZU2B5NGLySSWdUI%3D&st=2020-10-02T20%3A01%3A34Z&se=2020-10-03T04%3A11%3A34Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_10/outputs/model.pkl?sv=2019-02-02&sr=b&sig=VtpR50tazr7hcGBYA18ih2KaLFk91zCpeFmx880ok0Y%3D&st=2020-10-02T20%3A01%3A34Z&se=2020-10-03T04%3A11%3A34Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_5/outputs/model.pkl?sv=2019-02-02&sr=b&sig=vO06RJE5bQ6Fc19%2FKBmaqqsMRyZL24RqdIbJVQ06eYQ%3D&st=2020-10-02T20%3A01%3A35Z&se=2020-10-03T04%3A11%3A35Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_4/outputs/model.pkl?sv=2019-02-02&sr=b&sig=7xagjGdS3jvM3kf2AlC8N8iS6fhOZNnnkl7QFlB0Ybo%3D&st=2020-10-02T20%3A01%3A35Z&se=2020-10-03T04%3A11%3A35Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_5/outputs/model.pkl?sv=2019-02-02&sr=b&sig=vO06RJE5bQ6Fc19%2FKBmaqqsMRyZL24RqdIbJVQ06eYQ%3D&st=2020-10-02T20%3A01%3A35Z&se=2020-10-03T04%3A11%3A35Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_10/outputs/model.pkl?sv=2019-02-02&sr=b&sig=VtpR50tazr7hcGBYA18ih2KaLFk91zCpeFmx880ok0Y%3D&st=2020-10-02T20%3A01%3A34Z&se=2020-10-03T04%3A11%3A34Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_2/outputs/model.pkl?sv=2019-02-02&sr=b&sig=TM1T0V5C2xMB%2FaR6UxYzrk5%2FD%2BX1ZU2B5NGLySSWdUI%3D&st=2020-10-02T20%3A01%3A34Z&se=2020-10-03T04%3A11%3A34Z&sp=r
WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /azureml/ExperimentRun/dcid.AutoML_4f8c7904-0fc7-4be7-bf07-c0fbe51b8c60_5/outputs/model.pkl?sv=2019-02-02&sr=b&sig=vO06RJE5bQ6Fc19%2FKBmaqqsMRyZL24RqdIbJVQ06eYQ%3D&st=2020-10-02T20%3A01%3A35Z&se=2020-10-03T04%3A11%3A35Z&sp=r
/projappl/project_2003107/anaconda3/envs/azure_automl/lib/python3.7/site-packages/sklearn/base.py:197: FutureWarning: From version 0.24, get_params will raise an AttributeError if a parameter cannot be retrieved as an instance attribute. Previously it would return None.
  FutureWarning)
VotingEnsemble                                 0:04:26       0.7340    0.7336
Stopping criteria reached at iteration 12. Ending experiment.
WARNING - Received unrecognized parameter ensemble_download_models_timeout_sec
WARNING - Received unrecognized parameter ensemble_download_models_timeout_sec
WARNING - Received unrecognized parameter ensemble_download_models_timeout_sec
****************************************************************************************************
numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
Current status: BestRunExplainModel. Best run model explanations started
Current status: ModelExplanationDataSetSetup. Model explanations data setup completed
Current status: PickSurrogateModel. Choosing LightGBM as the surrogate model for explanations
Current status: EngineeredFeatureExplanations. Computation of engineered features started
Setting feature_perturbation = "tree_path_dependent" because no background data was given.
Current status: EngineeredFeatureExplanations. Computation of engineered features completed
Current status: BestRunExplainModel. Best run model explanations completed
****************************************************************************************************
WARNING - Received unrecognized parameter ensemble_download_models_timeout_sec
WARNING - Received unrecognized parameter ensemble_download_models_timeout_sec
WARNING - Received unrecognized parameter ensemble_download_models_timeout_sec
Best pipeline:
From version 0.24, get_params will raise an AttributeError if a parameter cannot be retrieved as an instance attribute. Previously it would return None.
0. ('timeseriestransformer', TimeSeriesTransformer(featurization_config=None,
                      pipeline_type=<TimeSeriesPipelineType.FULL: 1>))
1. ('StandardScalerWrapper', <azureml.automl.runtime.shared.model_wrappers.StandardScalerWrapper object at 0x7f3245b90950>)
2. ('LassoLars', LassoLars(alpha=0.001, copy_X=True, eps=2.220446049250313e-16,
          fit_intercept=True, fit_path=True, max_iter=500, normalize=False,
          positive=False, precompute='auto', verbose=False))

['horizon_origin', '_automl_target_col_lag1T', '_automl_target_col_min_window2T', '_automl_target_col_max_window2T', '_automl_target_col_mean_window2T', 'quarter', 'month', 'day', 'hour', 'minute', 'am_pm', 'hour12', 'wday', 'qday', 'yday']
       RawFeatureName TypeDetected Dropped  EngineeredFeatureCount  \
0  timestamp           DateTime     No      11                       
1  _automl_target_col  Numeric      No      4                        

                                                                                                                                                                                                                            Transformations  
0  [MaxHorizonFeaturizer, DateTimeTransformer, DateTimeTransformer, DateTimeTransformer, DateTimeTransformer, DateTimeTransformer, DateTimeTransformer, DateTimeTransformer, DateTimeTransformer, DateTimeTransformer, DateTimeTransformer]  
1  [Lag, minRollingWindow, maxRollingWindow, meanRollingWindow]                                                                                                                                                                              
The concatenated objects may have different ts_value_colname values, result using the ts_value_colname of the first object
Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=False'.

To retain the current behavior and silence the warning, pass 'sort=True'.

The concatenated objects may have different ts_value_colname values, result using the ts_value_colname of the first object
